docs/package.json

```diff
@@ -13,6 +13,7 @@
   "dependencies": {
     "@astrojs/sitemap": "^3.1.6",
     "@astrojs/starlight": "^0.26.3",
+    "astro": "^4.15.9",
```

Comment:
@swaingotnochill why's this change required?

Reply:
I had to install astro to run the docs site. It will not work on machines where astro is not installed.

Reply:
But is this change required to be checked in? Don't see this as part of other folks PR on docs.

Reply:
I had issues without astro as a dependency because it was not installed on my system. One of the question is if astro is installed globally, then i don't see any issues. But if you try to generate the docs on a system without astro, it should cause an issue. 

Can you test this locally on your end?

Reply:
Besides, shouldn't astro be a dependency for the docs site which is built using astro?

Reply:
Okay, so i was kinda right. We need astro which is obvious and this is getting pulled transitively from "@astrojs/starlight"
```
➜  docs git:(swaingotnochill/docs/keys) ✗ npm list astro
DiceDB@0.0.1 /Users/swaingotnochill/sources/source_files/dice/docs
├─┬ @astrojs/starlight@0.26.3
│ ├─┬ @astrojs/mdx@3.1.5
│ │ └── astro@4.15.3 deduped
│ ├─┬ astro-expressive-code@0.35.6
│ │ └── astro@4.15.3 deduped
│ └── astro@4.15.3
└─┬ lucide-astro@0.438.0
  └── astro@4.15.3 deduped
  ```
  Then it make sense to just let starlight select the correct astro version.